CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
-0.77.x (TBD)
+0.78.0 (TBD)
```

Comment:
Very minor nit, but shouldn't we do these in the release branch? Otherwise it's not clear who should do it, and it's not really relevant to any specific feature/bugfix.